static/js/app.js

```diff
@@ -217,7 +228,7 @@ window.saveit = function saveit(level, lang, name, code, cb) {
     if (! window.auth.profile) {
        if (! confirm (window.auth.texts.save_prompt)) return;
        // If there's an adventure_name, we store it together with the level, because it won't be available otherwise after signup/login.
-       if (window.State.adventure_name) level = [level, window.State.adventure_name];
+       if (window.State && window.State.adventure_name) level = [level, window.State.adventure_name];
```

Comment:
Isn't `saveit` only called from the code page, where `window.State` is always defined?

Reply:
🤷‍♂️ I had this fail on me